src/server.c

```diff
@@ -3566,7 +3566,7 @@ int processCommand(client *c) {
          * That is unless lua_timedout, in which case client may run
          * some commands. */
         serverAssert(!server.in_exec);
-        serverAssert(!server.in_script);
+        serverAssert(!scriptIsRunning());
```

Comment:
I believe this PR removes all usage of server.in_script, so can you also remove the global value from server.h and the value that is being set in script.c?

Reply:
Removed `in_script` everywhere